Server/JokesOnYou.Web.Api/Services/UserService.cs

```diff
@@ -25,7 +25,7 @@ public async Task DeleteUser(string id)
 
         public async Task<IEnumerable<User>> GetAll()
```

Comment:
Change name to GetAllAsync since its an async task. 
Could even change to GetAllUsersAsync to be more specific.